src/lib/FreeC/Backend/Agda/Syntax.hs

```diff
@@ -3,7 +3,71 @@
 
 module FreeC.Backend.Agda.Syntax
   ( module Agda.Syntax.Concrete
+  , module Agda.Syntax.Common
+  , module Agda.Syntax.Position
+  , name
+  , intLiteral
+  , simpleImport
+  , lambda
+  , qname
+  , qname'
```

Comment:
I usually try to organize the export list in the same way I've organized the rest of the module.

```suggestion
    -- * Identifiers
  , name
  , qname
  , qname'
    -- * Imports
  , simpleImport
    -- * Expressions
  , intLiteral
  , lambda
```